### PR TITLE
Task-56509 :added the mainApplicationName property when creating an activity-stream

### PR DESCRIPTION
**Pull request by Amine0008**
added the mainApplicationName property when creating an activity-stream so we can track the use of exo's main applications in mobile statistics

### CODE REVIEW

**Comment by ghost** [comment]
### CodeSee Review Map:
[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=1435&src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

Review [in an interactive map](https://app.codesee.io/r/reviews?pr=1435&src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

View [more CodeSee Maps](https://app.codesee.io/r/maps?src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

#### Legend
CodeSee Map Legend

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/main.js [hunk 1]
```diff
@@ -45,6 +45,9 @@ const urls = [
   `${eXo.env.portal.context}/${eXo.env.portal.rest}/i18n/bundle/locale.social.Webui-${lang}.json`,
 ];
 
+//used only in mobile statistics
+eXo.env.portal.mainApplicationName = 'Social';
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/main.js]:
```suggestion
eXo.env.portal.mainApplicationName = 'activityStream';
```

It's better to add a name that you can add in i18N File and that is significative. To know if you want to use `social` (one single module name for all social apps) or `activityStream` + `peopleList` + `spaceList` + (moduleName per Application) ... could you please ask @Samuel-Renault-eXo or Morgan

BTW, could you please define `analytics.field.label.APP_NAME` in each project inside the file `locale.portlet.Analytics_en.properties` to allow internationalize the labels. (Each addon define this same file, so you will have to add a specific label in each file for each addon)

**Comment by ahamdi** [comment]
@Amine0008 Could you check the review ?

**Review by ahamdi — APPROVED** [review]

**Review by azayati — CHANGES_REQUESTED** [review]
Please update the title in order to use a simple verb like Add not added. And for the description try to explain the problem and how it is fixed.

**Comment by boubaker** [comment]
The issue has been postponed to post TC, thus I close the issue to be worked in dedicated FT.

**Pull request closed without merging**